ods_ci/build/Dockerfile_interop

```diff
@@ -0,0 +1,54 @@
+FROM quay.io/centos/centos:stream9
+
+# Use this build arg to set any default test script arguments
+ENV RUN_SCRIPT_ARGS=${RUN_SCRIPT_ARGS}
+ENV ROBOT_EXTRA_ARGS=''
+ENV SET_ENVIRONMENT=0
+ENV RETURN_PW=0
+ENV OC_HOST=${OC_HOST}
+ENV RUN_FROM_CONTAINER=1
+ENV SE_BROWSER_PATH=/usr/bin/chromium-browser
+ARG OC_VERSION=4.13
```

Comment:
Shouldn't we merge this PR to `releases/2.13.0` then rather than master?
How are we to control ods-ci versioning for image builds?

Reply:
I agree with @omrirh and start with this in releases/2.13.0. The master branch is being updated for rhoai-2.16 changes

Reply:
As Lisa has suggested, for the testing purposes let's stay with the master branch